install.sh

```diff
@@ -17,6 +22,16 @@ debian_distribution_name=""
 sfx_ingest_url="https://ingest.signalfx.com"
 dimensions=""
 input_collectd=""
+# Number of seconds system clock can be out of sync from NTP server.
+time_sync_max_drift=10
+
+is_bsd() {
```

Comment:
I meant use in determine_os where we have already call
if [ "$(uname)" == "Darwin" ]; then

if we add freebsd support we will need to change that anyway as it might output FreeBSD